src/main/kotlin/com/safehill/kclient/models/users/RemoteUser.kt

```diff
@@ -30,17 +31,22 @@ data class RemoteUser(
         other as RemoteUser
 
         if (identifier != other.identifier) return false
+        if (name != other.name) return false
```

Comment:
It boils down to the emission of objects from StateFlows and compose States.... If we have a flow or state of RemoteUser and the name of it changes the flow is not gonna emit because it thinks its the same user due to equality comparison.

Reply:
Couldn't it have other side effects? If the purpose is to change the view of the name changes is there any other approach?

Reply:
I am not sure what other effects we may have. 
Two users should be equal if and only if all the parameters match.
If there is a case when we want only the identifiers to match, then we should specifically check for the equality of  identifiers.